gdshelpers/parts/waveguide.py

```diff
@@ -407,19 +408,31 @@ def add_route_single_circle_to(self, final_coordinates, final_angle, max_bend_st
 
         radius = min([max_bend_strength, max_poss_radius]) if max_bend_strength is not None else max_poss_radius
         d = abs(radius * np.tan(diff_angle / 2))
-
-        tmp_wg = Waveguide.make_at_port(self._current_port)
-        tmp_wg.add_straight_segment(distance[0] - d)
-        tmp_wg.add_bend(-diff_angle, radius)
+        if final_width is not None:
```

Comment:
If you include `final_width = final_width if final_width is not None else self.width` at the beginning of the function, we only need the first branch of the if-statement -> if we one day change something else, we won't have to update both branches